src/web-ui/src/public/CategoryDetail.vue

```diff
@@ -1,50 +1,61 @@
 <template>
-  <Layout>
+  <Layout :loading="!products.length">
     <div class="content">
 
-      <!-- Loading Indicator -->
-      <div class="container mb-4" v-if="!products.length">
-        <i class="fas fa-spinner fa-spin fa-3x"></i>
-      </div>
-
       <!-- Product List -->
       <div class="container mt-3" v-if="products.length">
-        <h4 class="text-left">{{ this.display | capitalize }}</h4>
-        <div v-if="explain_recommended" class="text-muted text-center">
+        <h2 class="text-left">{{ this.display | capitalize }}</h2>
+        <div v-if="explain_recommended" class="text-muted text-left">
           <small><em><i v-if="active_experiment" class="fa fa-balance-scale"></i><i v-if="personalized" class="fa fa-user-check"></i> {{ explain_recommended }}</em></small>
         </div>
-        <div class="row">
+        <div class="row mt-4">
 
           <div class="col-sm-3 col-md-3 col-lg-3 text-left">
-            <div class="card mb-3">
-              <div class="card-body">
-                <h5 class="card-title mb-0">Gender</h5>
+            <h4 class="bg-light p-2">Filters</h4>
+            <div class="mb-3 border-bottom">
+                <h5
```

Comment:
Let's make the collapsable anchors buttons instead to maintain semantic markup.

Also, the Gender anchor doesn't have enough margin-bottom currently. It looks too close to the divider when collapsed as a consequence.